### DISCUSSION TITLE
mbsync: Option to ignore changes from specified fields

### DISCUSSION THREAD

**Discussion by Maxr1998** [beetbox/beets#4260] [Ideas]
A new `--ignore` option could be added to the mbsync plugin, where a list of beets fields could be specified to ignore/exclude changes for.

In my current use case, it would allow me to ignore updates to track titles and prevent file name changes (which would cause my music player to re-index them as "new") while still obtaining updates for other metadata.

**Comment by Maxr1998** [comment]
Implementing this should be relatively straightforward, I believe, mainly changing the way `autotag.apply_item_metadata` works.

**Comment by wisp3rwind** [comment]
Not quite the same problem, but a somewhat related thought about fine-grained handling of metadata is https://github.com/beetbox/beets/issues/4087#issuecomment-932901710

Generally, is your usecase best solved by a CLI flag, or rather a permanent config option?

**Reply by Maxr1998** [reply]
Yup, that seems like a similar case, even if it's more generalized (per field) and not per album + field like I intended to use it.

I think a config option would be great for permanent overrides, like you say, but in my case a CLI option would be better. I only update metadata one album at a time, so I just need to ignore those fields once.

**Reply by Maxr1998** [reply]
I've noticed this discussion has the `needinfo` label applied, is there anything else you need?

**Reply by wisp3rwind** [reply]
> I've noticed this discussion has the `needinfo` label applied, is there anything else you need?

No, if I remember correctly, I added that label when asking

> Generally, is your usecase best solved by a CLI flag, or rather a permanent config option?

so I guess it can be removed.
